.github/workflows/wheels.yml

```diff
@@ -112,63 +119,47 @@ jobs:
-            GFORTRAN_LOC=$(brew --prefix gfortran)/bin/gfortran 
-            ln -s $GFORTRAN_LOC gfortran
-            export PATH=$PWD:$PATH
-            echo "PATH=$PATH" >> "$GITHUB_ENV"
-          
-            # location of the gfortran's libraries
-            GFORTRAN_LIB=$(dirname `gfortran --print-file-name libgfortran.dylib`)
-
-            CIBW="MACOSX_DEPLOYMENT_TARGET=12.0\
-              MACOS_DEPLOYMENT_TARGET=12.0\
-              LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH\
-              _PYTHON_HOST_PLATFORM=macosx-12.0-arm64\
-              PIP_PRE=1\
-              PIP_NO_BUILD_ISOLATION=false\
-              PKG_CONFIG_PATH=/opt/arm64-builds/lib/pkgconfig\
-              PIP_EXTRA_INDEX_URL=https://pypi.anaconda.org/scientific-python-nightly-wheels/simple"
-            echo "CIBW_ENVIRONMENT_MACOS=$CIBW" >> "$GITHUB_ENV"
-          
-            CIBW="sudo xcode-select -s /Applications/Xcode_15.2.app"
-            echo "CIBW_BEFORE_ALL=$CIBW" >> $GITHUB_ENV
-
-            echo "REPAIR_PATH=/opt/arm64-builds/lib" >> "$GITHUB_ENV"
-
-            CIBW="DYLD_LIBRARY_PATH=$GFORTRAN_LIB:/opt/arm64-builds/lib delocate-listdeps {wheel} &&\
-              DYLD_LIBRARY_PATH=$GFORTRAN_LIB:/opt/arm64-builds/lib delocate-wheel --require-archs {delocate_archs} -w {dest_dir} {wheel}"
-            echo "CIBW_REPAIR_WHEEL_COMMAND_MACOS=$CIBW" >> "$GITHUB_ENV"
-      
+            LIB_PATH=/opt/arm64-builds/lib
           else
-            # macosx_x86_64 with OpenBLAS
-            # setting SDKROOT necessary when using the gfortran compiler
-            # installed in cibw_before_build_macos.sh
-            # MACOS_DEPLOYMENT_TARGET is set because of
-            # https://github.com/mesonbuild/meson-python/pull/309. Once
-            # an update is released, then that environment variable can
-            # be removed.
-            CIBW="MACOSX_DEPLOYMENT_TARGET=10.9\
-              MACOS_DEPLOYMENT_TARGET=10.9\
-              SDKROOT=/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk\
-              LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH\
-              _PYTHON_HOST_PLATFORM=macosx-10.9-x86_64\
-              PIP_PRE=1\
-              PIP_NO_BUILD_ISOLATION=false\
-              PIP_EXTRA_INDEX_URL=https://pypi.anaconda.org/scientific-python-nightly-wheels/simple"
-            echo "CIBW_ENVIRONMENT_MACOS=$CIBW" >> "$GITHUB_ENV"  
-          
-            CIBW="DYLD_LIBRARY_PATH=/usr/local/lib delocate-listdeps {wheel} &&\
-              DYLD_LIBRARY_PATH=/usr/local/lib delocate-wheel --require-archs {delocate_archs} -w {dest_dir} {wheel}"
-            echo "CIBW_REPAIR_WHEEL_COMMAND_MACOS=$CIBW" >> "$GITHUB_ENV"
+            if [[ ${{ matrix.buildplat[3] }} == 'accelerate' ]]; then
+              if [[ ${{ matrix.buildplat[4] }} == 14.0 ]]; then
+                # the macos-13 image that's used for building the x86_64 wheel
```

Comment:
I suggested above that we build macos_x86_64 on the macos-14 image (even though it's an M1 processor). I think the tests should still be able to run. So remove L137 to L143 when you try that out.